connector_importer/components/importer_csv_std.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.addons.component.core import Component
+
+from ..log import logger
+
+
+class RecordImporterCSVStd(Component):
+    """CSV Standard importer for records.
+
+    This importer is used to import standard CSV files, using the `load()`
+    method of Odoo.
+    """
+
+    _name = 'importer.record.csv.std'
+    _inherit = ['importer.record']
+    _break_on_error = True  # We want the import to stop if an error occurs
+
+    @property
+    def mapper(self):
+        if not self._mapper:
+            self._mapper = self.component(usage='importer.mapper.csv.std')
+        return self._mapper
+
+    def prepare_load_params(self, lines):
+        """Prepare the parameters for the `load()` standard method.
+
+        It returns a list of fieldnames + the list of corresponding values.
+        """
+        fieldnames = list(lines[0].keys())
+
+        data = [
+            [line[fieldname] for fieldname in fieldnames]
+            for line in lines
+        ]
+        return fieldnames, data
+
+    def run(self, record, is_last_importer=True, **kw):
+        """Run record job.
+
+        Steps:
+
+        * for each record, check if it is already imported or not and reference
+          them as created or updated
+        * launch the import with 'load()' method
+        * analyse error messages returned by 'load()' and remove relevant
+          references from the first step + create log error for them
+        * produce a report and store it on recordset
+        """
+
+        self.record = record
+        if not self.record:
+            # maybe deleted???
+            msg = 'NO RECORD FOUND, maybe deleted? Check your jobs!'
+            logger.error(msg)
+            return
```

Comment:
It may be useful to return a message, it will be displayed in the job's result (if the job returns the result of `run()`).